spinnaker/copy_kube_config/remove_temp_user.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Refactored to use VMAccesLinuxExtention so that this script and add_temp_user script are no longer necessary. I also refactored to use python JSON parsing when getting the nameSuffix to make it more robust. @clguimanMSFT PTAL at these refactorings